apps/ledger-live-mobile/src/screens/Settings/Debug/Debugging/StorageMigration.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { useState } from "react";
+import styled from "styled-components/native";
+import { useTheme } from "@react-navigation/native";
+import Clipboard from "@react-native-clipboard/clipboard";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import { Flex, Alert, Text, Tag } from "@ledgerhq/native-ui";
+import NavigationScrollView from "~/components/NavigationScrollView";
+import Button from "~/components/Button";
+import type { Theme } from "~/colors";
+import storage from "LLM/storage";
+import type { StorageState } from "LLM/storage/types";
+import { MIGRATION_STATUS, ROLLBACK_STATUS } from "LLM/storage/utils/migrations/constants";
+import type { MigrationStatus, RollbackStatus } from "LLM/storage/utils/migrations/types";
+
+export function DebugStorageMigration() {
+  const { colors } = useTheme();
+  const protectFeature = useFeature("llmMmkvMigration");
+  const [storageState, setStorageState] = useState({ ...storage.getState() });
+
+  async function handleMigrateBtnPress() {
+    await storage.migrate();
+    setStorageState({ ...storage.getState() });
+  }
+
+  async function handleRollbackBtnPress() {
+    await storage.rollbackMigration();
+    setStorageState({ ...storage.getState() });
+  }
+
+  async function handleResetBtnPress() {
+    await storage.resetMigration();
+    setStorageState({ ...storage.getState() });
+  }
+
+  async function handleCopyBtnPress() {
+    const json = await storage.stringify();
+    Clipboard.setString(json);
+  }
+
+  if (protectFeature == null || !protectFeature.enabled) {
+    return null;
+  }
```

Comment:
I'll suggest to hide the row directly in Debug Screen. You can use `FeatureToggle` to do that